src/modules/partition/core/PartitionLayout.cpp

```diff
@@ -239,6 +241,26 @@ PartitionLayout::execute( Device* dev,
         {
             currentPartition->fileSystem().setLabel( part.partLabel );
         }
+        if ( !part.partFeatures.isEmpty() )
+        {
+            for ( const auto &k : part.partFeatures.keys() )
+            {
+                const auto &v = part.partFeatures.value(k);
+
+                switch (v.type())
+                {
+                case QMetaType::Bool: 
+                    currentPartition->fileSystem().addFeature( { k, v.toBool() } );
```

Comment:
Since `addFeature()` is a KPMCore 4.1.0 API addition, this should be properly `#ifdef`fed. Calamares is compatible with KPMCore 3 and 4.0 and .. so this can't be used just-like-that.

Reply:
Done.04